index.bs

```diff
@@ -514,6 +514,19 @@ See also:
 
 This section details design principles for features which are exposed via CSS.
 
+<h3 id="css-content-should-be-visible">Content should be readable by default</h3>
+
+Design CSS properties or CSS layout systems (which are typically values of the `display` property), 
+to preserve the content as readable and usable by default.  
```

Comment:
```suggestion
to preserve the content as perceivable and usable by default.  
```

Just being consistent. We don't need to go with this, though.

Reply:
Similar to the previous issue - this should read: "to preserve content as viewable, accessible and usable by default"